modules/claims_api/lib/bd/bd.rb

```diff
@@ -120,6 +120,15 @@ def generate_upload_body(claim:, doc_type:, pdf_path:, action:, file_number: nil
       payload[:file] = Faraday::UploadIO.new(pdf_path.to_s, 'application/pdf')
       payload
     end
+
+    def determine_birls_file_number(doc_type, auth_headers)
+      if %w[L122 L705].include?(doc_type)
+        birls_file_num = auth_headers['va_eauth_birlsfilenumber']
+      elsif %w[L075 L190].include?(doc_type)
```

Comment:
@mchristiansonVA  Getting some odd behavior here but I am just need some clarification.  First off,  I see L075 in both other these conditionals?

Secondly, it does not seem to be detecting as expected?

<img width="565" alt="Screenshot 2024-09-16 at 1 08 17 PM" src="https://github.com/user-attachments/assets/4dc4b3ff-0040-4009-ade7-3f895e670f2d">